src/basho_bench.app.src

```diff
@@ -84,9 +84,9 @@
         %%
         %% Value generators
         %%
-        %% {fixed_bin, N} - Fixed size binary blob of N bytes
+        %% Default value generator is random an interger
         %%
-        {value_generator, {fixed_bin, 100}},
+        {value_generator, {uniform_int, 1}},
```

Comment:
Isn't this always going to to generate the number `1`?

Reply:
yes, it always `1` in default. For the most part we need overwrite value in config file.

Reply:
If we always override it, may I ask what the intention is in changing this?

Reply:
my original purpose is that we can speed up to the worker initialization process(https://github.com/cloudant/basho_bench/blob/cloudant/src/basho_bench_worker.erl#L407) in here if we don't override the value_generator with modified default value of `uniform_int ` due to  `fixed_bin` will take a longer time to generate one 96M random binary data when worker id is `1`. if we make  `uniform_int ` as default will generator a value as fast as possible to initialized the worker process.